kernels/inix/nix-kernel/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv
+, python3Packages
+, python3
+, pkgs
+, fetchFromGitHub
+}:
+with python3.pkgs;
+python3Packages.buildPythonPackage rec {
+  pname = "nix-kernel";
+  version = "0.1";
+  src = fetchFromGitHub {
+    owner = "GTrunSec";
+    repo = "nix-kernel";
+    rev = "dfa42d0812d508ded99f690ee1a83281d900a3ec";
+    sha256 = "1lf4rbbxjmq9h6g3wrdzx3v3dn1bndfmiybxiy0sjavgb6lzc8kq";
+  };
```

Comment:
Why do you use a fork of the kernel?

Reply:
> Why do you use a fork of the kernel?

waiting for merge in. https://github.com/corps/nix-kernel/pull/1